channelserver/src/server.rs

```diff
@@ -12,11 +12,10 @@ use rand::{self, Rng, ThreadRng};
 use uuid::Uuid;
 
 use logging::MozLogger;
+use meta;
 use perror;
 use settings::Settings;
 
-pub const EOL:&'static str = "\x04";
```

Comment:
Is removal of this constant on purpose, or an artifact of a bad merge or similar?  ISTM it's better as a constant.

Reply:
It's from a bad merge.